tests/Doctrine/Tests/DBAL/Schema/MySqlSchemaManagerTest.php

```diff
@@ -15,7 +16,7 @@ class MySqlSchemaManagerTest extends \PHPUnit\Framework\TestCase
      */
     private $manager;
 
-    /** @var Connection */
+    /** @var Connection|MockObject */
```

Comment:
It looks like the opposite of https://github.com/doctrine/dbal/pull/3260/files#diff-90748caec80a98aa0fcfef640916661bL27.

Reply:
I see. Makes sense.

I thought, using `phpstan-phpunit` would eliminate the necessity to explicitly annotate mocks as such. Looks like it's only the case for the return values of methods like `createMock()` which automatically get the traits of the object under test and the mock. But for properties, you still have to specify the type explictly. Does it sound about right?

Reply:
Yes, it converts them in this case, currently hardcoded in PHPStan, but will be cleaner and part of the phpunit extension in 0.11.